include/seqan/sequence/string_set_concat_direct.h

```diff
@@ -139,6 +141,22 @@ class StringSet<TString, Owner<ConcatDirect<TSpec> > >
     }
 };
 
+// --------------------------------------------------------------------------
+// Functional Adder_ (used in second signature of append())
+// --------------------------------------------------------------------------
+
+struct Adder_ : public std::unary_function<unsigned, unsigned>
+{
+    unsigned plus;
+    Adder_(unsigned _plus) : plus(_plus) {}
+
+    template <typename TValue>
+    unsigned operator() (TValue const & val) const
+    {
+        return val + plus;
+    }
+};
+
```

Comment:
Do we still need this? I see that the corresponding code is commented. So this will not be used. If it will be used, please use a generic Size type instead of the unsigned. But if this is not needed anymore, please remove it.